goal_src/jak2/pc/progress/progress-static-pc.gc

```diff
@@ -55,18 +294,40 @@
                                         ;; NOTE - this doesn't actually work (naughty dog tried it too in their progress code)
                                         ;; fix it eventually
                                         (load-level-text-files (the-as int (-> *setting-control* user-default language)))))
-    (progress-new-generic-link (text-id progress-misc-game-options)
-      (progress-new-generic-scrolling-page (text-id progress-misc-game-options)
-        (new 'static 'menu-generic-boolean-option
-          :name (text-id progress-speedrunner-mode)
-          :truthy-text (text-id progress-on)
-          :falsey-text (text-id progress-off)
-          :get-value-fn (lambda () (-> *pc-settings* speedrunner-mode?))
-          :on-confirm (lambda ((val symbol))
-            (set! (-> *pc-settings* speedrunner-mode?) val)
-            (commit-to-file *pc-settings*)))))))
+    (progress-new-generic-link-to-scrolling-page (text-id progress-misc-game-options)
+      (new 'static 'menu-generic-boolean-option
+        :name (text-id progress-discord-rpc)
+        :truthy-text (text-id progress-on)
+        :falsey-text (text-id progress-off)
+        :get-value-fn (lambda () (-> *pc-settings* discord-rpc?))
+        :on-confirm (lambda ((val symbol))
+          (set! (-> *pc-settings* discord-rpc?) val)
+          (commit-to-file *pc-settings*)))
+      (new 'static 'menu-generic-boolean-option
+        :name (text-id progress-speedrunner-mode)
+        :truthy-text (text-id progress-on)
+        :falsey-text (text-id progress-off)
+        :get-value-fn (lambda () (-> *pc-settings* speedrunner-mode?))
+        :on-confirm (lambda ((val symbol))
+          (set! (-> *pc-settings* speedrunner-mode?) val)
+          (commit-to-file *pc-settings*))))))
 
 
+;; TODO - migrate this menu as well
+;; graphic options
+;;   resolution
+;;   display mode
+;;   display select (disable if windowed)
+;;   vsync
+;;   aspect ratio
```

Comment:
I think some of these are already there (vsync, full screen)

Reply:
Yeah, this is just the full jak 1 menu layout for reference.